src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,77 @@
+package view;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import domain.Ranking;
+import dto.LottoResultDto;
+import dto.LottoTicketDto;
+
+public class OutputView {
+
+	private static final String PURCHASE_MESSAGE = "개를 구매했습니다.";
+	private static final String LOTTO_RESULT_MESSAGE = "당첨 통계\n---------";
+
+	public static void printLottoTicket(LottoTicketDto lottoTicketDto) {
+		lottoTicketDto.getLottoNumbersDto()
+			.stream()
+			.forEach(System.out::println);
```

Comment:
```suggestion
		lottoTicketDto.getLottoNumbersDto()
			.forEach(System.out::println);
```
이건 어떨까요🧐? 그리고 stream().forEach()와 .forEach()의 차이는 뭘까요?

Reply:
반영하겠습니다. 감사합니다!

Steam의 foreach는 Collection의 foreach와 달리 거대한 자료구조의 병렬 프로그래밍에 특화된 설계를 가지고 있습니다. 
Stream의 foreach는 애초에 아주 거대한 컬렉션을 다루기 위한 Stream에서 사용하는 반복문으로 Lazy한 연산을 위한 Loop Fusion이 일어납니다.  그래서 eager한 결과를 기대한 채 사용한다면 문제가 발생할 수도 있을것 같습니다.

이런 기능들이나 병렬 프로그래밍 혹은 stream의 다른 기능들이 딱히 필요 없는 경우에는 그냥 Collection의 foreach를 호출하는 것이 적절해 보입니다.